lib/canvas/live_events.rb

```diff
@@ -184,6 +205,23 @@ def self.wiki_page_deleted(page)
     })
   end
 
+  def self.attachment_created(attachment)
+    post_event_stringified('attachment_created', get_attachment_data(attachment))
+  end
+
+  def self.attachment_updated(attachment, old_display_name)
+    payload = get_attachment_data(attachment)
+    if old_display_name
```

Comment:
Do you have a use case for needing the old_display_name, or were you just following the pattern of updated events?  One of the reasons this isn't public yet is that we've been debating whether or not to include these old_* type fields in final public api, and just asking people to track what changed themselves if they care about that.  I'd love to hear more about your use case here if you have one and how onerous it would be for you if it wasn't there.